superset/models/dashboard.py

```diff
@@ -414,13 +415,12 @@ def export_dashboards(  # pylint: disable=too-many-locals
                 "native_filter_configuration", []
             )
             for native_filter in native_filter_configuration:
-                session = db.session()
```

Comment:
It's unclear why a new session was instantiated (and never closed) here. On line #433 the Flask-SQLAlchemy session is used and thus it seems prudent (and hopefully) safe to use `db.session`.